spec/functions/crc32_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'stdlib::crc32' do
+  context 'when default' do
+    it { is_expected.not_to eq(nil) }
+    it { is_expected.to run.with_params.and_raise_error(ArgumentError, %r{stdlib::crc32}) }
+  end
+
+  context 'when testing a simple string' do
+    it { is_expected.to run.with_params('my string').and_return('18fbd270') }
+    it { is_expected.to run.with_params('0').and_return('f4dbdf21') }
+  end
+
+  context 'when testing a sensitive string' do
+    it { is_expected.to run.with_params(sensitive('my string')).and_return('18fbd270') }
+  end
+
+  context 'when testing an integer' do
+    it { is_expected.to run.with_params(0).and_return('f4dbdf21') }
+    it { is_expected.to run.with_params(100).and_return('237750ea') }
+    it { is_expected.to run.with_params(sensitive(100)).and_return('237750ea') }
+  end
+
+  context 'when testing a float' do
+    it { is_expected.to run.with_params(200.300).and_return('7d5469f0') }
```

Comment:
`200.300` and `200.3` describe the same source so I think Rubocop is correct here.

Reply:
I was intending to test that first the float rules of drop the trailing `00` are applied before the resource is converted to a string, but that may not be worth testing?

Reply:
I think it's changed during parsing:
```console
$ pry
[1] pry(main)> 200.300
=> 200.3
```